src/backend/TrafficCourts/Workflow.Service/Mappings/Mapper.cs

```diff
@@ -14,8 +14,17 @@ public static EmailHistory ToEmailHistory(DisputantEmailSent src)
         target.ToEmailAddress = src.Message?.To is not null ? src.Message.To : "unknown";
         target.SuccessfullySent = EmailHistorySuccessfullySent.Y;
         target.Subject = src.Message?.Subject is not null ? src.Message.Subject : "unknown";
-        target.HtmlContent = src.Message?.HtmlContent is not null ? src.Message.HtmlContent : "";
-        target.PlainTextContent = src.Message?.TextContent is not null ? src.Message.TextContent : "";
+        // ensure only one of html content or plain text content is set
+        if (src.Message?.HtmlContent is not null && src.Message?.TextContent is not null)
```

Comment:
I do not like having this logic at this layer. Emails can have both HTML and Plain Text. If the ORDS endpoint requires one or the other, we can move this logic further down the pipeline, like into the Oracle Data API. I believe at this point we are only sending plain text emails. We had a conversation with the product owner and using HTML emails add additional compilations for formatting.